src/Meshes/geometry.ts

```diff
@@ -672,6 +678,10 @@ export class Geometry implements IGetSetVerticesData {
 
         meshes.splice(index, 1);
 
+        if (this._vertexArrayObjects) {
+            mesh.invalidateInstanceVertexArrayObject();

```

Comment:
Same here. How do I know that this function exist? Should I maybe do a dummy function on Mesh that is overwritten by instancedMesh.ts, like _disposeInstanceSpecificData ?

Reply:
same